src/OrchardCore.Modules/OrchardCore.OpenId/ViewModels/OpenIdClientSettingsViewModel.cs

```diff
@@ -20,7 +20,7 @@ public class OpenIdClientSettingsViewModel
         [RegularExpression(@"\/[-A-Za-z0-9+&@#\/%?=~_|!:,.;]+[-A-Za-z0-9+&@#\/%=~_|]", ErrorMessage = "Invalid path")]
         public string CallbackPath { get; set; }
 
-        [Url(ErrorMessage = "Invalid signeout redirect url")]
+        [Url(ErrorMessage = "Invalid signedout redirect url")]
```

Comment:
While we're at discussing a single-character change :) I think it would be better as:

```suggestion
        [Url(ErrorMessage = "Invalid SignedOut redirect url")]
```

This is how it's written on the UI too.

Reply:
Ok, SignedOut.